src/views/dependencyExplorer.ts

```diff
@@ -35,7 +35,7 @@ export class DependencyExplorer {
         });
 
         this._dataProvider.onDidChangeTreeData(() => {
-            if (window.activeTextEditor && Settings.syncWithFolderExplorer()) {
```

Comment:
Shall we sync with editor when tree data changed (like change the representation of package) if `syncWithFolderExplorer` is false? If we keep this conditon, tree view will not refresh when user set `syncWithFolderExplorer` to true for `setting.js` will call refresh first and then update the dependencyConfig when configuration changed